### ISSUE TITLE
When creating a default app and publishing it I get {"message":"Missing Authentication Token"}

### ISSUE THREAD

**Issue by hortinstein** [awsm-org/awsm#16]
```
{"message":"Missing Authentication Token"}
```

I checked and the authentication options are all

```
     "AuthorizationType": "none",
      "ApiKeyRequired": false,
```

What am I missing?

**Comment by eahefnawy** [comment]
Yeah I remember I got that error too when I deployed/tested the lambda. There was a bug in my code causing the error. Unfortunately the error had nothing to do with authentication :). I guess AWS is not super clear about that.

I suggest you check the logs after you test the lambda. You'll find lots of clues there. Just keep in mind that it doesn't have to be an authentication issue.
